release-notes/docs/3.11.0.md

```diff
@@ -0,0 +1,201 @@
+# 3.11.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [Supported software](#supported-software)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+- [medic-android#127](https://github.com/medic/medic-android/issues/127): Image upload forms crash the app. This has been broken for some time and is not easy to fix while supporting Android 4.4 so the resolution has been deferred until we can make this breaking change. Reach out if you require this feature in the near future.
+
+## Upgrade notes
+
+- This upgrade can be rolled out remotely. Users will download the new version in the background and be prompted to reload the app when it's ready. A small amount of data will be needed to download the new version.
+- Upgrading to this version does not require other applications to be upgraded. This release does not drop support for any hardware or software that works with the previous version.
+- This release modifies some CouchDB view definitions so we recommend you Stage the upgrade and wait for the views to be rebuilt before rolling it out to minimize downtime.
+- It is recommended to test the upgrade on a clone of your server first to ensure it works well for your application.
+
+### Breaking changes
+
+#### Removal of Reporting Rates feature
+
+This feature wasn't documented or tested and hasn't been used in production for some time so it has been removed. To read more about this feature or if this breaking change affects your read this [forum post](https://forum.communityhealthtoolkit.org/t/feature-removal-proposal-reporting-rates/1053).
+
+- [cht-core#6248](https://github.com/medic/cht-core/issues/6248): Remove the Reporting Rates feature
+
+### UI/UX changes
+
+#### Android app
```

Comment:
Shouldn't all of this actually be in the medic-android release notes instead/as well?

Reply:
In future yes, but for 3.11.0 and before everything goes in the CHT Core release notes. I'm comfortable having it here for this one last release, if that's ok with you?

Reply:
It's just kind of weird because: we _do_ have [release notes for medic-android](https://github.com/medic/medic-android#080), there's duplicate information between the two, and there's actually more info here than there.  Perhaps either link from there to here, or vice versa?

Reply:
Though I also understand if we don't want to be linking around if files are going to be moving around. I'm OK not doing anything else here if you don't think it's worth it.

Reply:
I've added a link from medic-android release notes to this doc.